### PR TITLE
Add package singlem 0.13.2

### PR DESCRIPTION
**Pull request by aaronmussig**
Adds a new package: singlem

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by raivivek** [comment]
Looks like there's a package dependency `orator` that's not in `conda-forge`. You might have to add that package too before this one can be built and merged.

File: recipes/singlem/build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# install python libraries
+python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+# copy main singlem python script
+chmod +x bin/singlem
```
Comment (marcelm) [inline recipes/singlem/build.sh]: If this actually necessary? Upstream lists `bin/singlem` as a script that should be installed by `pip install`. If the missing executable bit is the problem, can you please report this upstream?

File: recipes/singlem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - nose >=1.0
```
Comment (marcelm) [inline recipes/singlem/meta.yaml]: nose can probably be removed from the host requirements

Comment (aaronmussig) [reply recipes/singlem/meta.yaml]: I can't get it to build without it, unfortunately.

File: recipes/singlem/build.sh (hunk 1 above)
Comment (aaronmussig) [reply recipes/singlem/build.sh]: Good point!

**Comment by aaronmussig** [comment]
> Looks like there's a package dependency `orator` that's not in `conda-forge`. You might have to add that package too before this one can be built and merged.

Seems so, I'll try get it in conda-forge then revisit this.

File: recipes/singlem/LICENSE [hunk 3]
```diff
@@ -0,0 +1,675 @@
+GNU GENERAL PUBLIC LICENSE
```
Comment (marcelm) [inline recipes/singlem/LICENSE]: Can you please request upstream that the license be included with the tarball? See https://bioconda.github.io/contributor/linting.html#gpl-requires-license-distributed

Comment (aaronmussig) [reply recipes/singlem/LICENSE]: Will do thanks for pointing that out.

**Comment by aaronmussig** [comment]
@BiocondaBot please fetch artifacts

**Comment by aaronmussig** [comment]
@BiocondaBot please fetch artifacts

**Comment by aaronmussig** [comment]
* Added maximum pandas version due to as_matrix() use, has been removed since 1.0.0.
* Needs [ExpressBetaDiversity](https://github.com/dparks1134/ExpressBetaDiversity/releases) on the path, a conda environment will be created for this

**Comment by aaronmussig** [comment]
* Not ready for merge yet, needs to have some issues with unit tests resolved.

**Comment by aaronmussig** [comment]
@BiocondaBot please fetch artifacts

**Comment by aaronmussig** [comment]
All unit tests are passing and is ready for merge.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**